service/track_post.go

```diff
@@ -0,0 +1,39 @@
+package service
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/bitrise-io/api-utils/httprequest"
+	"github.com/bitrise-io/api-utils/httpresponse"
+	"github.com/bitrise-io/bitrise-step-analytics/models"
+	"github.com/gofrs/uuid"
+	"github.com/pkg/errors"
+)
+
+func TrackPostHandler(w http.ResponseWriter, r *http.Request) error {
+	var trackAnalytics models.TrackEvent
+	defer httprequest.BodyCloseWithErrorLog(r)
+	if err := json.NewDecoder(r.Body).Decode(&trackAnalytics); err != nil {
+		return httpresponse.RespondWithBadRequestError(w, "Invalid request body, JSON decode failed")
+	}
+
+	if trackAnalytics.EventName == "" {
+		return httpresponse.RespondWithBadRequestError(w, "Invalid request body, please provide event's name")
+	}
+
+	if _, err := uuid.FromString(trackAnalytics.ID); err != nil {
+		return httpresponse.RespondWithBadRequestError(w, fmt.Sprintf("Invalid request body, provided id is not valid uuid: %s", trackAnalytics.ID))
+	}
+
+	tracker, err := GetTrackerFromContext(r.Context())
+	if err != nil {
+		return errors.WithStack(err)
```

Comment:
should not we write some response as well?

Reply:
This is done the same way for all the endpoints I kept it for consistency reasons. Do you think we need to add message?

Reply:
In this case, let's leave it as it is.